.typos.toml

```diff
@@ -0,0 +1,33 @@
+[files]
+extend-exclude = [
+    ".git/",
+    "tests/Fixtures/Integration/misc/",
+    "tests/Fixtures/Integration/priority/",
+]
+ignore-hidden = false
+
+[default]
+extend-ignore-re = [
+    "Sebastiaan Stok",
```

Comment:
Instead of listing Sebastiaan explicitly, maybe we can use `@author .*` and just ignore any contributor's name?

Reply:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/9901473b79fe1b8bb6c6d419e1014deabda74afe/tests/DocBlock/AnnotationTest.php#L591
I want to make sure Chuck Norris' name is never misspelled 🙃

Reply:
I just wouldn't want to maintain this file everytime new contributor provides new files with author that does not fit typo checker, especially there's no official way of running it locally (it's not a part of `compploser qa`).

Reply:
How many incident will we have? One typo-like named contributor per 5 years?

Reply:
> `typos` is written in Go and is a single binary 1️⃣ 

So maybe we should install a binary file in a Composer hook (post-autoload-dump) and allow running it locally? `composer typos` added to `composer static-analysis` would allow preventing failures in CI.

Reply:
I don't know how a developer's life goes.

Reply:
agree, let's treat it similar as `checkbashisms`